### PR TITLE
Imporoved UI a little

### PR DESCRIPTION
**Pull request by s0nskar**
#### Task Done
- Added modal for adding cats photos
- Added materialbox for viewing image (Click on image)
- Added grid logic

Any suggestions.
Current Images:

screenshot from 2016-06-24 03-28-06
screenshot from 2016-06-24 03-28-23
screenshot from 2016-06-24 03-29-46

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -27,18 +40,24 @@ $('.submit').on("click", function(event) {
   });
 
 function htmlDisplay(photo, id){
-  console.log(photo);
   var html = '';
-  html += '<div class="row"><div class="col s12 m7"><div class="card small">';
+  html += '<div class="col s3 m4 s12"><div class="card">';
     html += '<div class="card-image">';
-      html += '<img src="' + photo.url + '" />';
+      html += '<img class="materialboxed" src="' + photo.url + '" />';
```
Comment (ghost) [inline main.js]: Is this new class needed? It's not used in css...

**Comment by atjonathan** [comment]
By the way, i'm working on the `refactor` branch, where i'm setting up `gulp` and `sass` and rewriting the javascript. I'll merge this in later today, thanks :smile:

File: main.js (hunk 1 above)
Comment (s0nskar) [inline main.js]: yes it is needed http://materializecss.com/media.html

Comment (ghost) [inline main.js]: That's cool :smile:, similar but sexier than lightbox :wink:

**Comment by sanskar-modi** [comment]
tell me if i can do anything, i just switch form python backend to js and want to learn javascript as much as i can.

**Comment by atjonathan** [comment]
@sanskar-modi, cool i'll message you on gitter later.

File: index.html [hunk 2]
```diff
@@ -12,54 +12,61 @@
...
-	  </div>
+		<!-- Add Cat Modal Trigger -->
+		<a class="modal-trigger waves-effect waves-light btn" href="#addCatModal">Add Cat Photo</a>
 
-	  <div class="results"></div>
+		<!-- Add Cat Modal Structure -->
+		<div id="addCatModal" class="modal">
+			<div class="modal-content">
+				<h4>Add Cat Photo</h4>
+				<i>“Meow” means “woof” in cat.</i>
```
Comment (ghost) [inline index.html:24]: What's this for?

Comment (s0nskar) [inline index.html:24]: what addCatModal ??
thats add cat modal structure
take a look here http://materializecss.com/modals.html

Comment (ghost) [inline index.html:24]: @sanskar-modi, no what's `<i>“Meow” means “woof” in cat.</i>` :joy:

Comment (s0nskar) [inline index.html:24]: thats nothing :laughing: :laughing: 
you can remove it.

Comment (ghost) [inline index.html:24]: :joy:

**Comment by atjonathan** [comment]
@sanskar-modi, you have merge conflicts :disappointed:, i'm ready to merge as soon as these are sorted :smile:

**Comment by sanskar-modi** [comment]
Well i fixed that.

**Pull request merged**